documentation/docs/guides/serialization.md

```diff
@@ -79,9 +78,9 @@ If you want to bypass the automatic serialization, you can directly write to the
 func helloWorld(c fuego.ContextNoBody) (any, error) {
 	w := c.Response()
 	w.Header().Set("Content-Type", "text/plain")
-	w.Write([]byte("Hello, World!")) // Write directly to the response writer.
+	w.Write([]byte("Hello, World!"))                                      // Write directly to the response writer.
```

Comment:
Weird

Reply:
Nothing weird. 

Take a look here. The comments are now aligned in the way Go formatting does.

![Uploading Screenshot_20250126_135734_Firefox.jpg …]()

You can see the document here

https://github.com/go-fuego/fuego/blob/0ea78e695346c1cded3e407cf2cd005dc53bf73b/documentation/docs/guides/serialization.md#custom-response---bypass-return-type

The diff format is simply making it not obvious, and it may appear weird.

Reply:
Ok, very nice then!

Reply:
mdsf does nothing but calling the command you configure in mdsf.json

So code blocks are formatted with our usual formatters, nothing much